CONTRIBUTING.md

```diff
@@ -67,8 +68,19 @@ Files are automatically formatted with prettier.
 
 ## Release
 
+### Main version
+For the main version, go on master (`git checkout master`) and use:
+
+```sh
+yarn run release
+```
+
+### Maintenance version
+
+For the maintenance version, go on maintenance (`git checkout maintenance`) and use:
+
 ```sh
-npm run release
+yarn run release-maintenance
```

Comment:
Late to the party: I never managed to run a scrip with yarn that will use "npm publish" at some point. I got credentials not being read for publishing to npm. I believe this is still an issue (and that's why previous command was npm run release).

To me that's one of yarn drawbacks that could make us switch back to npm (because if we need to use both yarn and npm, it's a struggle)